### DISCUSSION TITLE
03/061-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B8%80%E7%AB%A0-%E9%81%81%E5%85%A5%E8%99%9A%E7%A9%BA.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#29] [General]
03/061-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B8%80%E7%AB%A0-%E9%81%81%E5%85%A5%E8%99%9A%E7%A9%BA.html
就物理学而言，瞬时星际通信中继网络（俗称 IIC）是执政体的的一大胜利，是人类技术水平和对自然掌握能力的有力证明。仅需要一个普朗克单位宽的，能够跨越星际的虫洞，就可用于比光速更快的通讯，这真是一个奇迹。
https://tts.liuli.moe/03/061-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%B8%80%E7%AB%A0-%E9%81%81%E5%85%A5%E8%99%9A%E7%A9%BA.html

**Comment by name8102** [comment]
这几章的翻译质量是难以接受的，由于我的中文和英文水平限制，这里我只提出一些物理学相关的修改意见。

> 旋转频率约为 300 赫兹，表面磁场仅为 10^4T，半径为 10 公里

此处的单位建议统一为中文或者字母

> 别忘了，当你离恒星越来越近时，辐射将与 r^2 成比例增加，并准备好接受小型无人驾驶飞机性能下降的影响。

这里的辐射不是与 $r^2$ 成比例而应该是 $r^{-2}$，原文中有一些歧义

> 脉冲星故障

pulsar glitch 应该翻译为脉冲星的自转突变，说一个自然天体有故障也太奇怪了

> 同样，r^2 也适用。

原文中为 $r^3$，指的是磁场强度会随距离的立方衰减。将脉冲星近似为磁偶极子也应该是 $r^3$

> 撞击本身几乎无足轻重，十亿吨级的光子很容易被中子甩掉，在超过一秒的时间内被潮汐力拉伸，从而撕裂船只。
The impact itself would hardly matter, gigaton blips easily shrugged off by neutronium, stretched out over a second or so by tidal shear tearing the ships apart.

这里光子被中子甩掉是不正确的，因为中子内部不含有光子，这种说法很奇怪。原文中的 blip 查到的含义有 something relatively small or inconsequential within a larger context，应该是船体的碎块；neutronium 指的是纯由中子构成的物质而不是中子本身，这里可能指的是中子星（不确定）。这句原文我也没读懂，但这个翻译肯定有问题。

> 异界物质

应为奇异物质，小说中虚构的用于超光速引擎的物质

>脉冲星会出故障

同上，glitch 应为自转突变

>当激活它的时候，阿姆斯特朗计划的星门组件的大小几乎是原来 Adept Blue 的 20 倍

Adept Bule 指的是蓝色大师空间站

> 但是对于所有的异界物质控制单元和 IIC 优化互连和排除锁定力场以及散布在结构上的场操纵驱动器来说

同上，奇异物质

> 这里不再是黑洞表面 {event horizon}

event horizon 有专门的翻译视界或者事件边境，指的是逃逸速度大于光速，无法脱离黑洞的地方。翻译成表面也不能说错，只是不太严谨。

> 其中矿场丢弃了储存的时空碎片，或者自爆从而引发了脉冲星故障

同上，glitch 应为自转突变

除了一些错误的翻译，这几章还有明显的语序不自然和严重的翻译腔。我的词汇量明显不足流畅阅读原文，中文水平也不能保证自行翻译的质量。如果有重新翻译这几章的计划，我愿意加入并提供一些物理学相关的帮助。

**Reply by rxliuli** [reply]
@andylizi @ArgusK17 @MADOKAAAAA 有人感兴趣一起重新校对这些章节么？还建议将这些专有名词添加到文档中维护，ref: [译名对照表](https://github.com/liuli-moe/to-the-stars/blob/master/books/99/010-%E8%AF%91%E5%90%8D%E5%AF%B9%E7%85%A7%E8%A1%A8.md)

如果是明显的错误建议直接直接提 PR 修改，正如之前所说，吾辈也并非译者，大部分时候只是提供技术性的支持，以发布 epub/website，但明显的错误仍然可以直接修改。ref: [发现的译文问题修改参考](https://github.com/liuli-moe/to-the-stars/issues/12)

@name8102 他们可能最近在忙翻译最新的第 66 章，回复可能会有延迟，ref: [第四卷翻译进度跟踪](https://github.com/liuli-moe/to-the-stars/issues/9)
